registry/registry_mock_test.go

```diff
@@ -32,31 +30,6 @@ func init() {
 
 	testHTTPServer = httptest.NewServer(handlerAccessLog(r))
 	testHTTPSServer = httptest.NewTLSServer(handlerAccessLog(r))
-
-	// override net.LookupIP
-	lookupIP = func(host string) ([]net.IP, error) {
-		if host == "127.0.0.1" {
-			// I believe in future Go versions this will fail, so let's fix it later
-			return net.LookupIP(host)
-		}
```

Comment:
As we're now skipping `lookupIP` if it's an IP address, which case can be removed; let me update